apps/prairielearn/public/stylesheets/local.css

```
@@ -369,6 +369,10 @@ small .user-output-invalid {
   background: #fff;
 }
 
+.btn.btn-light.text-danger {
```

Let's leave a note about why we're doing this:

```suggestion
/**
 * We use red text in light buttons to indicate errors on many elements.
 * By default, this has insufficient contrast. We override the color in
 * this specific case to improve the contrast.
 */
.btn.btn-light.text-danger {
```